pkg/kubelet/volume/volume_manager.go

```diff
@@ -185,10 +187,17 @@ type volumeManager struct {
 	// desiredStateOfWorldPopulator runs an asynchronous periodic loop to
 	// populate the desiredStateOfWorld using the kubelet PodManager.
 	desiredStateOfWorldPopulator populator.DesiredStateOfWorldPopulator
+
+	// mountedVolulmeToClean is the list of volumes already mounted before
+	// kubelet starts. They need to be cleaned up before reconciler runs
+	mountedVolumeToClean []operationexecutor.MountedVolume
 }
 
 func (vm *volumeManager) Run(stopCh <-chan struct{}) {
 	defer runtime.HandleCrash()
+	glog.Infof("Clean up already mounted volumes before Kubelet starts")
```

Comment:
Saad and I discussed a little bit about it, it seems no good way to identify Orphaned volumes clearly when kubelet starts. During sync loop, Although there are some orphaned volume message, we don't want to do unmount them for safe. Need more discussion about it. Please let me know any thoughts you might have. Thanks!